test/dotnet.Tests/dotnet-msbuild/GivenDotnetRunInvocation.cs

```diff
@@ -23,8 +23,6 @@ public GivenDotnetRunInvocation(ITestOutputHelper log)
         [InlineData(new string[] { "-p", "prop1=true", "-p", "prop2=false" }, new string[] { "--property:prop1=true", "--property:prop2=false" })]
         [InlineData(new string[] { "-p:prop1=true;prop2=false" }, new string[] { "--property:prop1=true", "--property:prop2=false" })]
         [InlineData(new string[] { "-p", "MyProject.csproj", "-p:prop1=true" }, new string[] { "--property:prop1=true" })]
-        // The longhand --property option should never be treated as a project
-        [InlineData(new string[] { "--property", "MyProject.csproj", "-p:prop1=true" }, new string[] { "--property:MyProject.csproj", "--property:prop1=true" })]
```

Comment:
Maybe this test could be moved so that this is a different failure, like "you said `--property` but didn't specify one"?